generators/chipyard/src/main/scala/config/IbexConfigs.scala

```diff
@@ -0,0 +1,14 @@
+package chipyard
+
+import chisel3._
+
+import freechips.rocketchip.config.{Config}
+
+// ---------------------
+// Ibex Configs
+// ---------------------
+
+class IbexConfig extends Config(
+  new chipyard.config.WithBootROM ++               // Ibex reset vector is at 0x80
+  new ibex.WithNIbexCores(1) ++                    // single Ibex core

```

Comment:
To confirm... this has all the default params that the Ibex core normally comes with (ex. 4 PMP registers, etc)?

Reply:
Yes, I have the correct default values for the parameters in the actual tile code.